pq-crypto/sike_r3/sikep434r3_fpx.c

```diff
@@ -0,0 +1,459 @@
+// -----------------------------------------------------------------------------
```

Comment:
Yes, I deleted/disabled the assembly portions. [For example this](https://github.com/aws/s2n-tls/blob/main/pq-crypto/sike_r3/sikep434r3_fpx.c#L186-L191).

Reply:
Also to reply to about replacing randombytes, to my understanding it is a s2n specific function which lives in a non pq-crypto directory within s2n. So I used RAND_bytes to avoid this dependency.

Reply:
I understand that, my question was why not do this:
`void randombytes() { RAND_bytes(); }` instead of changing all the calls.

Reply:
Oh I see. I am not sure if this matters but I thought style wise it would make more sense to see aws-lc functions than something like. I can definitely see why that would be easier though. Please let me know if there is a preference!